### ISSUE TITLE
Support for TypeScript labeled tuples not working

### ISSUE THREAD

**Issue by sunyatasattva** [Automattic/calypso-prettier#33]
I tried using the latest `wp-prettier@2.2.1-beta-1` version with a type that looks like this: 

```ts
type RangeValue = [ lowerBound: number, upperBound: number ]; 
```

My Prettier configuration uses `typescript` as parser. Nevertheless, this throws an error:

```
SyntaxError: ',' expected.
```

This, however, correctly works in the latest Prettier version: <kbd>[Prettier playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEMCeAHOACASgQygHM4A1fAGwFccBebAbWwogHc4AnAIQiqgBMk2KFQC2AI04AabFQxZuvAUJETO2ALoBubCCkgIGGAEtoAZ2Sh8HDmwAK1hBZSVW+NBf3iO+MAGs4GABlfFE4ABljKDhkADNKMzgvH39AoIxfKKJkGA4afTg1fn44fnDCIip8EgAxCA5RfBgTYmQQfCoYCD0QAAsYUQoAdV7jeDMMsDggpzHjADcxtDawM08QKMSOGDsfIka4hKSQACszAA8grIo4AEUqCHhDikT9DI4ttvQsMzAOYyMPQw-1gQ2M-BgvWQAA4AAxvWyJIY+DBtYFwLbzGL6ACOD3gu0MznaZgAtNFSqUehw4HjjDTdtUDkh4i9jolRMYcnl2dc7viYiyjvoYPhxGCIVCkAAmEU+YwULIAYQgomZIAxAFYelREgAVMXOVmvEDzGgASQECGCfwBMAAggIgugbs8TWdzlwUgFgqEIlFBca4ABfYNAA)</kbd> and is correctly parsed in AST as `TSNamedTupleMember`. It seems support for this was added by Prettier in 2.1.0 ([issue link](https://github.com/prettier/prettier/issues/8646), [blog post](https://deploy-preview-8932--prettier.netlify.app/blog/2020/08/24/2.1.0.html)).

**Comment by jsnajdr** [comment]
Hi @sunyatasattva :wave: I just updated the fork to the latest upstream version, 2.6.2, and published it to NPM as `wp-prettier@2.6.2-beta-1`, the the labeled tuples formatting works there. Please try it out.

**Comment by sunyatasattva** [comment]
Sounds great, thank you!
